tests/foreman/installer/test_installer.py

```diff
@@ -182,9 +182,9 @@ def install_satellite(satellite, installer_args, enable_fapolicyd=False):
     )
 
 
-def setup_capsule_repos(satellite, capsule_host, org, ak):
+def sync_capsule_repos(satellite, capsule_host, org, ak):
     """
-    Enables repositories that are necessary to install capsule
+    On Satellite enables and synchronizes content that is necessary to install capsule
```

Comment:
I would suggest using the [imperative mood](https://docs.astral.sh/ruff/rules/non-imperative-mood/).

```suggestion
    Enable and synchronize content on Satellite required for Capsule installation.
```